src/database/migrations/20191031090043-status-accommodation.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Accommodations','status',{
```

Comment:
this should also be updated to `isAvailable` Boolean datatype, `allowNull false`, and default to `true`